run/GCHP/runScriptSamples/operational_examples/harvard_cannon/gchp.submit_consecutive_jobs.sh

```diff
@@ -8,7 +8,7 @@
 # One argument: number of runs
 numRuns=${1}
 if [[ "x${numRuns}" == "x" ]]; then
-    echo "ERROR: Specify number of runs as argument, e.g. ./gchp.multirun.submit 2"
+    echo "ERROR: Specify number of runs as argument, e.g. ./gchp.submit_consecutive_jobs.sh 2"
     exit
 fi
 
```

Comment:
I think line 17  below also needs to be changed from:

```
echo "Duration: $(grep "DURATION=" setCommonRunSettings.sh | cut -c 8- | xargs)"
```

 to:

```
 echo "Duration:  $(grep "Run_Duration=" setCommonRunSettings.sh | cut -c 14- | xargs)"
 ```

Reply:
Good catch. All set.